### PR TITLE
Fix : Responsive Accordion #49

### PR DESCRIPTION
**Pull request by Im-Abhi**
Issue Related : #49
Preview 

Accodion_Preview

### CODE REVIEW

**Comment by Im-Abhi** [comment]
@nandini7637 Please review the PR for the same . Thank you

**Review by nandini7637 — CHANGES_REQUESTED** [review]
1- The hover color for small devices is black so if you could change that, it will be great
2- Increase font of the dopdown content.
3- Remove the readme of progress bar

else everything looks nice
good work!

File: Accordion/Responsive_Accordion_01/code/style.css [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+    width: 80%;
+    min-width: 150px;
+    height: 100%;
+    margin: 0 auto;
+    overflow-y: scroll;
+    background-color: #FFF;
+    font-family:  sans-serif;
+    color: rgba(48, 69, 92, 0.8);
+  }  
+  .accordion-bral {
```
Comment (Ovenoboyo) [inline Accordion/Responsive_Accordion_01/code/style.css:11]: Indentation here seems weird. For guidelines you can refer to [this](https://google.github.io/styleguide/htmlcssguide.html)

File: Accordion/Responsive_Accordion_01/code/style.css [hunk 2]
```diff
@@ -0,0 +1,127 @@
+html {
```
Comment (Ovenoboyo) [inline Accordion/Responsive_Accordion_01/code/style.css:1]: Also avoid directly styling html tags. If possible please move this to a css class

**Review by Ovenoboyo — CHANGES_REQUESTED** [review]

**Comment by Im-Abhi** [comment]
@nandini7637 @Ovenoboyo  I have completed the requested changes please review them. Thank you

**Review by Ovenoboyo — APPROVED** [review]

**Review by Abirpal202049 — APPROVED** [review]

**Review by nandini7637 — APPROVED** [review]

**Review by thepranaygupta — APPROVED** [review]
LGTM 👍

**Review by saikatmu31 — APPROVED** [review]

**Pull request merged**